src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -160,10 +160,24 @@
             <argument type="service" id="sylius.context.channel" />
         </service>
 
-        <service id="Sylius\Bundle\ApiBundle\Serializer\AddProductReviewCommandFieldItemIriToIdentifierDenormalizer">
+        <service id="Sylius\Bundle\ApiBundle\Map\CommandItemIriArgumentToIdentifierMap">
+            <argument type="collection">
+                <argument key="Sylius\Bundle\ApiBundle\Command\AddProductReview">product</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\Checkout\ChoosePaymentMethod">paymentMethod</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\Account\ChangePaymentMethod">paymentMethod</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\RequestResetPasswordToken">locale</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\ResendVerificationEmail">locale</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\Cart\PickupCart">locale</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\Cart\AddItemToCart">productVariant</argument>
+                <argument key="Sylius\Bundle\ApiBundle\Command\Checkout\ChooseShippingMethod">shippingMethod</argument>
+            </argument>
```

Comment:
That's a neat idea, but I'm afraid about this list expandability. One would have to override this service definition, therefore they will not receive changes to this collection that we will do in the later releases. We need some refactoring here. Probably configuration tree would be the best choice. Maybe second class for client-specific mappings?